components/sidebar/sidebar.jsx

```diff
@@ -340,7 +340,7 @@ export default class Sidebar extends React.PureComponent {
                 nextIndex = curIndex - 1;
             }
             const nextChannel = allChannelIds[Utils.mod(nextIndex, allChannelIds.length)];
-            this.props.actions.goToChannelById(nextChannel);
+            goToChannelById(nextChannel);
```

Comment:
Though untouched by this change, I feel like we should be using `nextChannelId` instead of `nextChannel`.
Non-blocking but good to update.